acquia_lift.admin.wizard.inc

```diff
@@ -1321,6 +1498,20 @@ function acquia_lift_personalize_campaign_wizard_review_alter(&$form, &$form_sta
  */
 
 /**
+ * Ajax submit handler for any help section dismissal button
+ */
+function acquia_lift_personalize_campaign_wizard_help_ajax($form, &$form_state) {
+  global $user;
+  $current_dismiss = !empty($user->data['acquia_lift_help_dismiss']) ? $user->data['acquia_lift_help_dismiss'] : array();
+  // #name property is "display_{step}"
+  $dismiss = preg_replace('/^dismiss_/', '', $form_state['triggering_element']['#name']);
+  if (!in_array($dismiss, $current_dismiss)) {
+    $current_dismiss[] = $dismiss;
+    user_save($user, array('data' => array('acquia_lift_help_dismiss' => $current_dismiss)));
```

Comment:
I might be reading this wrong... couldn't we be clobbering other stuff in the `$user->data` property here?